### ISSUE TITLE
Change of inline elements

### ISSUE THREAD

**Issue by rgieseke** [stencila/encoda#902]
Thanks for the merge of https://github.com/stencila/encoda/pull/896

Is the general workaround/change in
https://github.com/stencila/encoda/pull/896/commits/1ab835ea4eae8a35432e3fd7e85354e104b02242#diff-fb6bbbe1b1c1c64434466345b13efc9495ced4ade665e4f015b3b40dc1471f33L1308
 
reasonable? Or is there a better approach?
This currently breaks install as i've only applied it to the JATS decoder.

**Comment by nokome** [comment]
Yes, that fix is the right one, but there are also other changes needed as a result of changes to how coercion to the schema is now activated for `Article.content`. I have tried to explain that in this PR https://github.com/stencila/encoda/pull/904.

My aim is to get that finished and do a release today.

**Comment by nokome** [comment]
I have merged #94 but there are still some weird results due to Ajv's weird coercion rule e.g. zero length string -> `null` and empty string (e.g. ' ') -> `0`. I will fix those as soon as possible and certainly before next release.
